cmd/doracled/cmd/get_oracle_key.go

```diff
@@ -78,7 +81,14 @@ func getOracleKeyCmd() *cobra.Command {
 				return errors.New("the existing node key is different from the one used in oracle registration. if you want to re-request RegisterOracle, delete the existing node_priv_key.sealed file and rerun register-oracle cmd")
 			}
 
-			return getOraclePrivKey(conf, oracleRegistration, nodePrivKey)
+			// TODO: This will be changed to the nonce that is field of oracleRegistration
```

Comment:
This change is reflected in panacea-core. (https://github.com/medibloc/panacea-core/pull/404)
Modify it using the master branch of panacea-core.

Reply:
BTW, I think we need to add 'Nonce' field in `NewMsgRegisterOracle` in [x/oracle/type/message_oracle.go](https://github.com/medibloc/panacea-core/blob/2c0f80b0f08e2ba3db98593ddeb384931618dae2/x/oracle/types/message_oracle.go#L10).

Reply:
Yes, I created this PR.(https://github.com/medibloc/panacea-core/pull/405)